Source/WebCore/PAL/pal/SessionID.h

```diff
@@ -73,36 +73,15 @@ class SessionID {
     friend bool operator==(SessionID, SessionID) = default;
     bool isAlwaysOnLoggingAllowed() const { return !isEphemeral(); }
 
-    template<class Encoder> void encode(Encoder&) const;
-    template<class Decoder> static std::optional<SessionID> decode(Decoder&);
-
     SessionID isolatedCopy() const { return *this; }
 
     explicit operator bool() const { return m_identifier; }
 
-private:
     static bool isValidSessionIDValue(uint64_t sessionID) { return sessionID != HashTableEmptyValueID && sessionID != HashTableDeletedValueID; }
-
```

Comment:
I think you can avoid making it public by using 
`friend struct IPC::ArgumentCoder<SessionID, void>;` below `private:`